pkg/core/native/designate.go

```diff
@@ -412,10 +420,20 @@ func (s *Designate) DesignateAsRole(ic *interop.Context, r noderoles.Role, pubs
 		return fmt.Errorf("failed to update Designation role data cache: %w", err)
 	}
 
-	ic.AddNotification(s.Hash, DesignationEventName, stackitem.NewArray([]stackitem.Item{
+	notification := stackitem.NewArray([]stackitem.Item{
```

Comment:
s/notification/ntf ?

I saw it several times before in previous PRs, let's agree to use short names for local function variables (if it's possible and if the variable meaning can easily be guessed from the short name). Long names are suitable for exported functions, exported package variables/constants and etc. to give some more context to the user.